3rd/CMakeLists.txt

```diff
@@ -225,7 +225,7 @@ if(NOT wxSQLite3_FOUND OR NOT (WXSQLITE3_HAVE_CODEC OR MMEX_ENCRYPTION_OPTIONAL)
     endif()
 
     if(NOT MSVC) # wxSQLite3 security ext doesn't require C 99 for MSVC
-        if(";i386;i686;amd64;x86_64;" MATCHES ";${CMAKE_HOST_SYSTEM_PROCESSOR};")
+        if(";i386;i686;amd64;x86_64;arm64;" MATCHES ";${CMAKE_HOST_SYSTEM_PROCESSOR};")
```

Comment:
Adding `aarch64` to the list of architectures nullifies the patch that was put in place for addressing https://github.com/moneymanagerex/moneymanagerex/issues/5953

The idea is that passing `-msse4.2 -maes` will cause aarch64 builds on Linux to fail. That's why the flatpak build patches CMakeLists to get multi-arch support:

https://github.com/flathub/org.moneymanagerex.MMEX/blob/e1f3b4f4f61a5360ed5d50f4b043a4b7a6683a13/cmakelists-3rd.patch

Reply:
A better solution may be to try to add `-msse4.2 -maes` *unless* it detects that it's building for aarch64 on Linux.